astro.config.mjs

```diff
@@ -0,0 +1,9 @@
+import netlify from '@astrojs/netlify' // todo: replace with vercel?
+import {defineConfig} from 'astro/config'
+
+export default defineConfig({
+	adapter: netlify(),
```

Comment:
I'd copied this from a netlify project I was working on. I think there's a Vercel one we can swap in, but I might need your help getting that configured.